yt_dlp/extractor/nhk.py

```diff
@@ -67,12 +76,23 @@ def get_clean_field(key):
         }
         if is_video:
             vod_id = episode['vod_id']
+            api_url, api_token = self._get_vod_api(vod_id)
+            streams_json = self._download_json(api_url, vod_id, query={
+                'token': api_token,
+                'type': 'json',
+                'optional_id': vod_id,
+                'active_flg': 1,  # dont know what this is but site sends it
```

Comment:
seems equally mysterious with or without comment
```suggestion
                'active_flg': 1,
```